app-thunderbird/src/daily/kotlin/net/thunderbird/android/featureflag/TbFeatureFlagFactory.kt

```diff
@@ -25,6 +26,7 @@ class TbFeatureFlagFactory : FeatureFlagFactory {
             FeatureFlag(MessageListFeatureFlags.UseComposeForMessageListItems, enabled = false),
             FeatureFlag(MessageViewFeatureFlags.ActionExportEml, enabled = true),
             FeatureFlag(AccountSettingsFeatureFlags.EnableAvatarCustomization, enabled = false),
+            FeatureFlag(MessageViewerFeatureFlags.UseNewMessageViewerCssStyles, enabled = false),
```

Comment:
You could enable features that are already functionally complete for daily builds as well.